components/automate-ui/src/app/modules/infra-proxy/create-org-modal/create-org-modal.component.html

```diff
@@ -60,7 +60,7 @@ <h2 slot="title">Add Chef Organization</h2>
           </label>
           <chef-error
             *ngIf="(createForm.get('admin_user').hasError('required') || createForm.get('admin_user').hasError('pattern')) && createForm.get('admin_user').dirty">
-            Admin user is required.
+            Admin User is required.
```

Comment:
Author note: Here we do want capitals because it is the name of the input field on screen (which occurs a few lines higher in the file).